spark/app/utils.py

```diff
@@ -0,0 +1,41 @@
+from . import spark
+from . import config
+
+class CSVFile(object):
+
+    def __init__(self,name):
+        self.name = name
+        bucket = config.get('AWS', 'bucket')
+        self.path = 's3a://{}/{}'.format(bucket, self.name)
+    def load(self):
+        print(self.path)
+        csv = spark.read.csv(self.path, header=True, inferSchema=True)
+        #df = spark.read.format('csv').options(header='true', inferSchema='true').load("s3a://wind-toolkit-csv/b'20070122140000'.csv")
```

Comment:
either remove or make load() work with both csvs and dfs depending on parameter input